src/Components/Calendar/Calendar.js

```diff
@@ -1,67 +1,126 @@
 import React, { Component } from 'react';
 import { View, StyleSheet, Button, Text } from 'react-native';
-import { connect } from 'react-redux'
 import CalendarStrip from 'react-native-calendar-strip';
-import { addDateToStore, loadRoutines } from '../../Actions/index'
+import { connect } from 'react-redux'
+import Dimensions from 'Dimensions';
+import moment from 'moment'
+// 
+import { RalewayText, RalewayBoldText } from '../../Utilities/RalewayText';
+import { setDate, loadRoutines, setSemanticDate } from '../../Actions/index'
 import { fetchRoutines } from '../../Thunks/fetchRoutines';
-import Container from '../Container/Container';
+
+
 
 export class Calendar extends Component {
 
+  componentDidMount = () => {
+    let originalYear = JSON.stringify(moment().year());
+    let originalMonth = JSON.stringify(moment().month() + 1);
+    let originalDay = JSON.stringify(moment().date());
+    this.determineDate(originalYear, originalMonth, originalDay)
+  }
+
   grabDate = (date) => {
     let slicedDate = JSON.stringify(date).slice(1, 11);
-    this.props.addDateToStore(slicedDate);
+    this.determineDate(slicedDate.slice(0, 4), slicedDate.slice(5, 7), slicedDate.slice(8, 10))
+    this.props.setDate(slicedDate);
     this.props.fetchRoutines(slicedDate);
   }
 
+  determineDay = (day) => {
+    let dayNum = day.split('')[0] === '0' ? day.slice(1, 2) : day;
+    let letters = [ 'th', 'st', 'nd', 'rd', 'th', 'th', 'th', 'th', 'th', 'th' ];
+    let lastDigit = day.split('').reverse()[0];
+    return `${dayNum}${letters[lastDigit]}`;
+  }
+
+  determineMonth = (month) => {
+    let monthNum = month.split('')[0] === '0' ? month.slice(1, 2) : month;
+    let months = [ 'January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
```

Comment:
Right now, when May shows up, it appears as Mav cause it gets cut off.  We should fix this with a bit more padding, or just enlarge the box.